src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -163,12 +165,28 @@ never arise that the same ID is hashed against the same data, and are
 introduced as if we expect them to be two separate IDs.
 
 Of course, with a malicious actor upstream, collisions are possible and can be
-engineered quite trivially (for example, just by returning a constant ID for all
-rows), but we're not supposing that the input `IDataView` is maliciously
+engineered quite trivially (for example, just by returning a constant ID for
+all rows), but we're not supposing that the input `IDataView` is maliciously
 engineering hash states, or applying the operations above in any strange way
-to attempt to induce collisions. For example, you could take, operation 1, define it
-to be the enumeration of all `UInt128` values, then take operation 2 to select
-out specifically those that are hash states that will result in collisions.
-But I'm supposing this is not happening. If you are running an implementation
-of a dataview in memory that you're supposing is malicious, you probably have
-bigger problems than someone inducing collisions.
\ No newline at end of file
+to attempt to induce collisions. For example, you could take, operation 1,
+define it to be the enumeration of all `DataViewRowId` values, then take
+operation 2 to select out specifically those that are hash states that will
+result in collisions. But I'm supposing this is not happening. If you are
+running an implementation of a dataview in memory that you're supposing is
+malicious, you probably have bigger problems than someone inducing collisions.
+
+Despite the apparent complexity, it should be noted that in *most*
+applications, since the vast majority of `IDataView` implementations are
```

Comment:
>applications [](start = 0, length = 12)

Should this be something like this?
"Despite the apparent complexity, it should be noted that in *most* applications it is actually not complicated, since the vast majority of `IDataView` ...."

Reply:
Yes, we can certainly word smith it better than this, doesn't make much sense... I think it would be better if I broke this sentence up a bit.